flask_resty/testing.py

```diff
@@ -28,11 +29,11 @@ def get_meta(response):
 
 def assert_value(actual, expected):
     if isinstance(expected, dict):
-        assert isinstance(actual, dict)
+        assert isinstance(actual, Mapping)
         for k, v in expected.items():
             assert_value(actual.get(k, None), v)
-    elif isinstance(expected, list):
-        assert isinstance(actual, list)
+    elif isinstance(expected, Sequence) and not isinstance(expected, String):
```

Comment:
No, I mean, why do `elif isinstance(expected, Sequence) and not isinstance(expected, String):` then? Why not just do `isinstance(expected, (list, tuple))` or something? I think you can also just catch the `TypeError` coming from the `zip_longest` as well in the list case.

The dict case looks fine, though.